ferrocene/doc/user-manual/src/rustfmt/install.rst

```diff
@@ -0,0 +1,37 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Installing ``rustfmt``
+======================
+
+1. Download the ``rustfmt`` archive for your host platform
+
+   The pattern of the name is:
+
+   .. code-block::
+
+      rustfmt-<target>-<hash>.tar.xz
```

Comment:
This is not always true. For stable releases we use the version number rather than the hash.